### ISSUE TITLE
EBM model what is the scale of the local explanations?

### ISSUE THREAD

**Issue by tmontana** [interpretml/interpret#157]
Hi all.  Just wondering how we should interpret the local explanations score?  How does it relate to the predicted probability score?  is there a method you'd recommend to transform the local explanation score into a percentage of probability?  What does an intercept of -2.5 mean?

In the example workbook for EBM the local explanations range from -2.5 to +2.5 give or take.

Many thanks

**Comment by interpret-ml** [comment]
Hi @tmontana, 

In regression, the local explanation scores are in the units of your output. For example, if you are predicting housing price in us dollars, each local explanation score will correspond to how many dollars that feature added or subtracted from the final prediction. 

In classification, the local explanation scores are logits, or log odds: (https://en.wikipedia.org/wiki/Logit). To convert these logits into a probability, we simply sum them up and pass them through the logistic link function: 1 / (1 + e^(-sum(logits))) (https://en.wikipedia.org/wiki/Logistic_function). For reference, this is how logistic regression works, and the scores are interpreted in the exact same way. As an example, here's how a variety of different summed logits translate into probabilities:

```python3
summed_logits = [-5, -2, -1, -0.5, 0, 0.5, 1, 2, 5]
for logit in summed_logits:
    probability = 1 / (1 + np.exp(-logit))
    print(logit, probability)
```
```python3
-5    0.006
-2    0.119
-1    0.269
-0.5  0.378
 0    0.5
 0.5  0.622
 1    0.731
 2    0.881
 5    0.993
```

It's important to note the non-linearity here -- starting at a score of 0 and adding a +2 score moves the probability from 0.5 to 0.881, but adding another +3 only moves it from 0.881 to 0.993. This non-linearity is why we show scores in additive (log_odds) space, where you can compare them, and not in probability space. 

Hope this helps!
-InterpretML Team

**Comment by tmontana** [comment]
Hi InterpretML.  A big thank you for taking the time to explain this.  It's very clear and very helpful.  Cheers!

**Comment by NogaG** [comment]
Hi all,
is it possible to show the explanations in probability terms and not logits?
